### PR TITLE
Cherrypicking 32b249a (#21710) into v11.5.0. 

### PR DESCRIPTION
**Pull request by blackdevelopa**
## **Description**

Cherrypicking [32b249a](https://github.com/MetaMask/metamask-extension/commit/32b249a506453d0fefb19d32cef27f46c4c55530) (https://github.com/MetaMask/metamask-extension/pull/21710) into v11.5.0. Commit fix [here](https://github.com/MetaMask/metamask-extension/commit/32b249a506453d0fefb19d32cef27f46c4c55530)

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/pages/settings/experimental-tab/experimental-tab.component.js [hunk 1]
```diff
@@ -9,9 +9,6 @@ import {
   MetaMetricsEventCategory,
   MetaMetricsEventName,
 } from '../../../../shared/constants/metametrics';
-///: BEGIN:ONLY_INCLUDE_IN(build-main)
-import { showSnapAccountExperimentalToggle } from '../../../../shared/modules/snap-accounts';
```
Comment (seaona) [inline ui/pages/settings/experimental-tab/experimental-tab.component.js:13]: any hints why are we changing all these snaps code here? Seems unrelated to the Terms and Conditions from Blockaid

**Comment by seaona** [comment]
There's some code changing the snaps area, which seems to have an undesired effect: now the Snaps option in Experimental tab has disappeared (see right side). Is this expected?

Screenshot from 2023-11-09 10-30-34

File: ui/pages/settings/experimental-tab/experimental-tab.component.js (hunk 1 above)
Comment (blackdevelopa) [reply ui/pages/settings/experimental-tab/experimental-tab.component.js]: TY. Great catch. Omitted this while resolving git conflict. Fixed https://github.com/MetaMask/metamask-extension/pull/21769/commits/d5e35eff672872a49c97389477c4758132b3d26b

**Comment by blackdevelopa** [comment]
Thank you @seaona, fixed. Should show this as expected now

Screenshot 2023-11-09 at 11 45 24

**Review by segun — APPROVED** [review]

**Comment by seaona** [comment]
looks good @blackdevelopa thank you!

**Review by seaona — APPROVED** [review]
